develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Connect to TiDB with mysql2 in AWS Lambda Function
+summary: This article describes how to build a CRUD application using TiDB and mysql2 in AWS Lambda Function and provides a simple example code snippet.
+---
+
+# Connect to TiDB with mysql2 in AWS Lambda Function
+
+TiDB is a MySQL-compatible database, [AWS Lambda Function](https://aws.amazon.com/lambda/) is a compute service, and [mysql2](https://github.com/sidorares/node-mysql2) is a popular open-source driver for Node.js.
+
+In this tutorial, you can learn how to use TiDB and mysql2 in AWS Lambda Function to accomplish the following tasks:
+
+- Set up your environment.
+- Connect to your TiDB cluster using mysql2.
+- Build and run your application. Optionally, you can find sample code snippets for basic CRUD operations.
+
+> **Note**
+>
+> This tutorial works with TiDB Serverless and TiDB Self-Hosted.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- [Node.js **18**](https://nodejs.org/en/download/) or later.
+- [Git](https://git-scm.com/downloads).
+- A TiDB cluster.
+- An [AWS account](https://repost.aws/knowledge-center/create-and-activate-aws-account).
+- An AWS user with access to the Lambda function.
```

Comment:
If we need to use `sam deploy`, we may also need permissions involving `CloudFormation`, `S3`, `APIGateway`:

```suggestion
- An [AWS user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) with administrator permissions.
```